### PR TITLE
Feature/9 1 medical staff home front

### PR DESCRIPTION
**Pull request by saksofon997**
Personal page for nurses, front + back. Tested. Managing nurses from AdminC dashboard front. Also tested

### CODE REVIEW

File: client/src/app/services/user.service.ts [hunk 1]
```diff
@@ -108,7 +108,10 @@ export class UserService {
 						} else if (this.checkLoggedIn().roles.indexOf('DOCTOR')!= -1){
 							console.log('HERE')
 							this.router.navigate(['/doctor']);
-						}else{
+						} else if (this.checkLoggedIn().roles.indexOf('NURSE')!= -1){
+							console.log('HERE')
```
Comment (coperope) [inline client/src/app/services/user.service.ts]: AHAHAH koji copy paste

File: server/src/main/java/com/project/tim49/controller/NurseController.java [hunk 2]
```diff
@@ -22,6 +24,31 @@
...
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    @PreAuthorize("hasAuthority('NURSE')")
+    public ResponseEntity getNurse(@PathVariable Long id) {
+        UserDTO patient = nurseService.findById(id);
+        return new ResponseEntity<>(patient, HttpStatus.OK);
+    }
+
+    @PutMapping(path="/change" )
+    @PreAuthorize("hasAuthority('NURSE') or hasAuthority('ADMINC')")
+    public ResponseEntity changeDoctor(@RequestBody NurseDTO nurseDTO){
```
Comment (coperope) [inline server/src/main/java/com/project/tim49/controller/NurseController.java]: sto se zove changeDoctor?

File: client/src/app/services/user.service.ts (hunk 1 above)
Comment (saksofon997) [reply client/src/app/services/user.service.ts]: Hahahahahahah, tezak zivot

File: server/src/main/java/com/project/tim49/controller/NurseController.java (hunk 2 above)
Comment (saksofon997) [reply server/src/main/java/com/project/tim49/controller/NurseController.java]: Promaklo mi ahahah

Comment (coperope) [reply server/src/main/java/com/project/tim49/controller/NurseController.java]: Hocemo sad menjati ili nek ide na master? 
Meni svejedno samo da ne zaboravimo kasnije, zbunjivace nas

Comment (coperope) [reply server/src/main/java/com/project/tim49/controller/NurseController.java]: Ipak merge ajde pa cemo sad u sl sprintu popravljati sve

**Pull request merged**